### PR TITLE
build(diff-app): add 'describeEach' chips

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ *  Convert the `describeEach` test title format into a list of more readable chips.
+ *  Chips are displayed differently depending on the param they represent:
+ *  - If param is boolean, show a green/red chip with the param name;
+ *  - If param is undefined, show nothing;
+ *  - Else, show a default chip with "key = value";
+ */
+@customElement('app-describe-each-chip-list')
+export class DescribeEachChipList extends LitElement {
```
Comment (kyubisation) [inline src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:31]: Nitpick; I would prefer a more noun-based name for this, e.g. `ChipListDescription`. What do you think?

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  public static override styles: CSSResultGroup = style;
+
+  @property()
+  public set testCaseName(name: string) {
+    this._chips = Array.from(name.matchAll(paramsRegex)).map(this._mapToChip);
+    this.requestUpdate();
```
Comment (kyubisation) [inline src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:37]: Is the requestUpdate necessary?

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts (hunk 2 above)
Comment (TomMenga) [reply src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:37]: I discovered that it's still necessary if you do not have a getter for the property :/

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts (hunk 1 above)
Comment (TomMenga) [reply src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:31]: To me `describe-each` was a noun (that indicated the describe-each pattern we have) but I agree that it looks ugly. I'd like to reference that it's a decomposition of the test tile, something like `TestTitleChipList`?

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts (hunk 2 above)
Comment (kyubisation) [reply src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:37]: We could just add the getter, but this is fine too.

File: src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts (hunk 1 above)
Comment (kyubisation) [reply src/visual-regression-app/src/components/test-case/describe-each-chip-list/describe-each-chip-list.ts:31]: Yeah, that sounds good 👍

**Pull request merged**